proxyui/src/main/webapp/WEB-INF/views/profiles.jsp

```diff
@@ -186,84 +152,88 @@
                   }
             });
             profileList.jqGrid('gridResize');
+
+            $('#configurationUploadForm').submit(function(event) {
+                event.preventDefault();
+
+                var file = $('#configurationUploadFile').get(0).files[0];
+                var formData = new FormData();
+                formData.append('fileData', file, file.name);
+
+                $.ajax({
+                    type: "POST",
+                    url: '<c:url value="/api/backup"/>',
+                    data: formData,
+                    processData: false,
+                    contentType: false,
+                    beforeSend: function() {
+                        $("#statusNotificationText").text("Uploading configuration...");
+                        $("#statusNotificationStateDiv")
+                            .removeClass("ui-state-error")
+                            .addClass("ui-state-highlight");
+                        $("#statusNotificationDiv").fadeIn();
+
+                        // disable form buttons
+                        $("#configurationUploadDialog ~ .ui-dialog-buttonpane button")
+                            .prop("disabled", true)
+                            .addClass("ui-state-disabled");
+                    },
+                    success: function() {
+                        window.location.reload();
+                    },
+                    error: function() {
+                        $("#statusNotificationStateDiv")
+                            .removeClass("ui-state-highlight")
+                            .addClass("ui-state-error");
+                        $("#statusNotificationText").text("An error occurred while uploading configuration...");
+
+                        // enable form buttons
+                        $("#configurationUploadDialog ~ .ui-dialog-buttonpane button")
+                            .prop("disabled", false)
+                            .removeClass("ui-state-disabled");
+                    }
+                });
+            });
         });
 
 
         function exportConfiguration() {
-            downloadFile('<c:url value="/api/backup"/>');
+            download('<c:url value="/api/backup"/>');
         }
 
         function importConfiguration() {
             $("#configurationUploadDialog").dialog({
                 title: "Upload New Configuration",
                 modal: true,
-                position:['top',20],
                 buttons: {
                   "Submit": function() {
                     // submit form
-                    $("#configurationUploadFileButton").click();
+                    $("#configurationUploadForm").submit();
                   },
                   "Cancel": function() {
                       $("#configurationUploadDialog").dialog("close");
                   }
                 }
             });
         }
-
-        window.onload = function () {
```

Comment:
moved under `$(document).ready`. Also switched to use `$.ajax` instead of vanilla XHR.